cmd/internal/cli/startvm.go

```diff
@@ -72,10 +72,10 @@ func startVM(sifImage, singAction, cliExtra string, isInternal bool) error {
 		sylog.Fatalf("Failed to determine image absolute path for %s: %s", sifImage, err)
 	}
 	if _, err := os.Stat(bzImage); os.IsNotExist(err) {
-		sylog.Fatalf("This functionality is not supported. For more information visit: https://sylabs.io/apptainer-desktop-macos")
+		sylog.Fatalf("This functionality is not supported. For more information visit: https://sylabs.io/singularity-desktop-macos")
```

Comment:
We should probably make a task to remove Singularity Desktop support.

Reply:
That's in #52